buildpacks/php/src/main.rs

```diff
@@ -53,18 +55,23 @@ impl Buildpack for PhpBuildpack {
         if loader.detect(&context.app_dir) {
             DetectResultBuilder::pass().build()
         } else {
+            print::bullet("PHP detection");
             loader_notices
                 .into_iter()
                 .map(PhpBuildpackNotice::ProjectLoader)
                 .for_each(notices::log);
-            log_info("No PHP project files found.");
+
+            print::sub_bullet("No PHP project files found.");
```

Comment:
If we're not doing bullet_stream in detect failure output: To avoid importing `libherokubuildpack::log::log_info` [the .NET buildpack just write a plain message with `println`](https://github.com/heroku/buildpacks-dotnet/blob/ab0cbd1cd8cfba0811d781d5e6bd82c20c031ea7/buildpacks/dotnet/src/main.rs#L55-L56), and then make sure to flush stdout after.

Maybe worth doing something like that here as well? E.g. like:

```suggestion
            println!(""No PHP project files found.");
            std::io::stdout().flush().expect("Couldn't flush output stream");
```